### ISSUE TITLE
Add PACER client code support to Juriscraper

### ISSUE THREAD

**Issue by mlissner** [freelawproject/juriscraper#192]
This would be really nice to add because it would allow people to see how we're using their credentials. For example, if we do bulk data with somebody's account, they'll know it was us because of the client code. 

Right now, none of the PACER crawling code supports this, so we should figure out how to add it. I think it's just a matter of adding it to our session during login.

**Comment by j2kao** [comment]
I actually had trouble logging into a PacerSession without specifying `login:clientCode` -- ended up downloading a separate repo and hardcoding that field into `PacerSession.login()`. Not sure if it is a recent/known issue?

**Comment by mlissner** [comment]
Hm, that's news to me. I wonder, are you using an institutional account? I wonder if there's a way to make the client code required? Can you log in to the PACER website leaving the client code blank? 

Would you be interested/willing to contribute a PR to add the client code to the PacerSession object? At a minimum, I suppose we could just make it accept it in the init and then use it in the login method. If we wanted to be clever, we could also add a method like, `change_client_code`, that could be used to change the client code (this is something the website lets you do, but I've never played with the GET/POST requests it sends).

**Comment by johnhawkinson** [comment]
Yes, PAA allows mandatory PACER client codes. Cf., among other things:
https://www.pacer.gov/psc/faq.html
<img width="749" alt="screen shot 2018-11-26 at 12 47 56" src="https://user-images.githubusercontent.com/1270718/49032044-98b2ed80-f179-11e8-8981-66bad64a1d06.png">

**Comment by mlissner** [comment]
Ah ha. I had a feeling that rang a bell. Thanks @johnhawkinson .

**Comment by j2kao** [comment]
Ah, thanks for the info, all.

Happy to do a PR at some point for this -- probably the simple version for now. :-)
